### PR TITLE
Adding a cron-jobs for testing SRO on OCP 4.9 and 4.8.

### PR DESCRIPTION
**Pull request by ybettan**
Signed-off-by: Yoni Bettan <ybettan@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml [hunk 1]
```diff
@@ -117,6 +117,104 @@ tests:
...
           memory: 500Mi
     workflow: generic-claim
+- as: e2e-aws-ocp-4.9-periodic
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 40m0s
+    version: "4.9"
```
Comment (pacevedom) [inline ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml:127]: This is going to test the latest SRO (which is 4.10) with OCP 4.9, shouldnt this be with 4.10?

Comment (ybettan) [reply ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml:127]: There is no support for OCP 4.10 ATM.

File: ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml [hunk 2]
```diff
@@ -117,6 +117,51 @@ tests:
...
     workflow: generic-claim
+- as: e2e-aws-ocp-4.8-periodic
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 40m0s
+    version: "4.8"
+  cron: 0 */24 * * *
```
Comment (qbarrand) [inline ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml:128]: Is there a reason why you use `*/24`? How about `0 4 * * *`?
`@daily` would be even better if it is supported. Unfortunately the [reference docs](https://steps.ci.openshift.org/ci-operator-reference) don't say that.

Comment (ybettan) [reply ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml:128]: > Is there a reason why you use */24? How about 0 4 * * *?

What is the difference between the 2?

I see that assisted is using `'@daily` https://github.com/ybettan/release/blob/98baa4c0db1e828de7439c118e83883904714e98/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml#L290

**Pull request merged**